Solution rebuilds every time APSIM is started from VS.

Every time you hit the Start button in VS (or press F5) the entire solution rebuilds even if nothing has changed since the last start. This slows debugging considerably as you have to wait about 40 seconds for the build to complete.

The issue seems to be related to CopyLocal: 
1>Project 'Models' is not up to date. CopyLocal reference source 'C:\apsimxtest\packages\ExcelDataReader.2.1.2.3\lib\net45\Excel.dll' is more recent than 'C:\apsimxtest\Bin\Excel.dll'.

Copying the DLL manually just results in another DLL giving the same issue. Have the dependencies been rearranged or something like that?

Mine is slow, but not 40 seconds...
1>------ Build started: Project: Models, Configuration: Debug Any CPU ------
1>  Models -> D:\Projects\ApsimX\Bin\Models.exe
2>------ Build started: Project: Importer, Configuration: Debug Any CPU ------
2>  Importer -> D:\Projects\ApsimX\Models\Bin\Importer.exe
3>------ Build started: Project: UserInterface, Configuration: Debug Any CPU ------
3>  UserInterface -> D:\Projects\ApsimX\Bin\UserInterface.exe
========== Build: 3 succeeded, 0 failed, 2 up-to-date, 0 skipped ==========

Greg's is much slower - didn't put a watch on it, but it's output is different again..
1>------ Build started: Project: Models, Configuration: Debug x86 ------
2>------ Build started: Project: SWIMFrame, Configuration: Debug x86 ------
2>  SWIMFrame -> D:\Apsim\ApsimX\Bin\SWIMFrame.exe
1>  Models -> D:\Apsim\ApsimX\Bin\Models.exe
3>------ Build started: Project: Importer, Configuration: Debug x86 ------
3>  Importer -> D:\Apsim\ApsimX\Models\Bin\Importer.exe
4>------ Build started: Project: UserInterface, Configuration: Debug x86 ------
5>------ Build started: Project: ApsimNG, Configuration: Debug x86 ------
4>  UserInterface -> D:\Apsim\ApsimX\Bin\UserInterface.exe
6>------ Build started: Project: UnitTests, Configuration: Debug x86 ------
6>  UnitTests -> D:\Apsim\ApsimX\Tests\UnitTests\bin\Debug\UnitTests.dll
5>D:\Apsim\ApsimX\ApsimNG\Views\ExplorerView.cs(209,45,209,63): warning CS0067: The event 'ExplorerView.ShortcutKeyPressed' is never used
5>D:\Apsim\ApsimX\ApsimNG\Views\GridView.cs(38,57,38,68): warning CS0067: The event 'GridView.ButtonClick' is never used
5>  ApsimNG -> D:\Apsim\ApsimX\Bin\ApsimNG.exe
========== Build: 6 succeeded, 0 failed, 3 up-to-date, 0 skipped ==========

I didn't see any references to CopyLocal on either machine - is that coming up through the build output Justin?

--- a/UserInterface/UserInterface.csproj
+++ b/UserInterface/UserInterface.csproj
@@ -139,14 +139,6 @@
       <HintPath>..\packages\MarkdownDeep.NET.1.5\lib\.NetFramework 3.5\MarkdownDeep.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="MathNet.Numerics, Version=3.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\MathNet.Numerics.3.0.0\lib\net40\MathNet.Numerics.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="MathNet.Numerics.Data.Text, Version=3.1.1.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\MathNet.Numerics.Data.Text.3.1.1\lib\net40\MathNet.Numerics.Data.Text.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
     <Reference Include="MigraDoc.DocumentObjectModel-gdi">
       <HintPath>..\packages\PDFsharp-MigraDoc-gdi.1.50.3638-beta\lib\net20\MigraDoc.DocumentObjectModel-gdi.dll</HintPath>
     </Reference>

